docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
 
 ## Overview
 
+Once your data is cleaned, you will need to get it out of OpenRefine and into the system of your choice. OpenRefine outputs a number of file formats, can upload your data directly into Google Sheets, and can export directly to Wikidata. 
+
+You can also export your full project data so that it can be opened by someone else using OpenRefine (or yourself, on another computer).
 
 ## Export data
 
 Note you will only export data in the current view - that is, with current filters and facets applied. 
```

Comment:
This isn't completely accurate. Some of the exports have the option to ignore current filters and facets